pkg/config/config_linux.go

```diff
@@ -1,7 +1,27 @@
 package config
 
+import (
+	"io/ioutil"
+	"strings"
+)
+
+func isWSL() bool {
+	data, err := ioutil.ReadFile("/proc/sys/kernel/osrelease")
```

Comment:
@shinhs0506 is this a large file? I wanna get an idea of the cost of reading it at startup

Reply:
The file is one line long

just as a reference, https://github.com/microsoft/WSL/issues/423#issuecomment-221627364

my `/proc/sys/kernel/osrelease` has the following content
`5.10.102.1-microsoft-standard-WSL2`

Reply:
sounds good!